src/Commands/ExportPostmanCommand.php

```diff
@@ -273,6 +290,55 @@ public function makeRequest($route, $method, $routeHeaders, $requestRules)
         return $data;
     }
 
+    /**
+     * Process a rule set and utilize the Validator to create human readable descriptions
+     * to help users provide valid data.
+     *
+     * @param $attribute
+     * @param $rules
+     * @return string
+     */
+    protected function parseRulesIntoHumanReadable($attribute, $rules): string
+    {
+        if (! $this->config['rules_to_human_readable']) {
+            return is_array($rules) ? implode(', ', $rules) : $rules->__toString();
```

Comment:
This is assuming that the `$rules` is a `Rule` class. This would throw an error if it was not for some reason.